src/validators/date_validator.coffee

```diff
@@ -1,20 +0,0 @@
-FormRenderer.Validators.DateValidator =
```

Comment:
These validators aren't gone; instead, they've been moved into the `validateType` method in the new field models to be closer to the logic they're validating.